docs/release.sh

```diff
@@ -78,6 +78,11 @@ upload_current_documentation() {
 	src=site/
 	dst=s3://$BUCKET$1
 
+	cache="max-age=3600"
+	if [ "$NOCACHE" ]; then
+		cache="no-cache"
```

Comment:
If you want me to review with my super pedantic hat on, this (and the one two lines above) are the ones I was referring to where double quotes aren't actually necessary. :+1: